nservicebus/sagas/message-correlation.md

```diff
@@ -12,7 +12,9 @@ related:
 
 Correlation is needed in order to find existing saga instances based on data on the incoming message. In the example the `OrderId ` property of the `CompleteOrder` message is used to find the existing saga instance for that order.
 
-To declare this use the `ConfigureHowToFindSaga` method and use the `Mapper` to specify to which saga property each message maps to. Note that NServiceBus will only allow correlation on a single saga property and that the property types must match.
+To declare this use the `ConfigureHowToFindSaga` method and use the `Mapper` to specify to which saga property each message maps to.
+
+NOTE: NServiceBus will only allows message mappings that all correlate to a single saga property and that all property types must match. Use a [custom saga finder](saga-finding.md). to correlate different properties and/or types.
```

Comment:
This is going to result in two back-to-back NOTE boxes with the partial below.

I'm also not sure it's wise to recommend custom saga finders this far up the page and highlighted so much. We should be going out of our way to downplay custom saga finders and instead educating on what to do instead.

Reply:
I'm with @DavidBoike on this one, to me personally it seems the previous version was clearer. Also we mention saga finders later, so why spell it out here so explicitly?
Was that change driven by a customer @ramonsmits ?